build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19.0 AS go_builder
+FROM golang:1.19.1 AS go_builder
```

Comment:
@hors maybe we should use `FROM golang:1.19`

Reply:
@hors fixed